tern/analyze/default/container/image.py

```diff
@@ -23,10 +24,30 @@
 logger = logging.getLogger(constants.logger_name)
 
 
+def download_container_image(image_tag_string):
+    '''Download the docker image and convert it into oci format'''
+    try:
+        # extract the docker image
+        docker_image = image_tag_string
+        if 'docker://' in image_tag_string:
+            docker_image = image_tag_string.split('docker://')[1]
+        image_attr = general.parse_image_string(docker_image)
+        oci_image = 'oci://{0}/{1}'.format(
+            rootfs.working_dir, image_attr.get('name'))
+        docker_image = 'docker://{0}'.format(docker_image)
```

Comment:
I just found out that if we use `docker://` rather than `dockerdaemon` we encounter a TLS error on the registry side. I also found that [crane](https://github.com/google/go-containerregistry/) is a much more lightweight tool to pull container images, and conforms with OCI's on-disk layout spec. @rnjudge Do you want to merge this PR and then let @mukultaneja update it to use crane?